README.md

```diff
@@ -1,6 +1,6 @@
 # [remind101/pkg](https://github.com/remind101/pkg) [![Build Status](https://travis-ci.org/remind101/pkg.svg?branch=master)](https://travis-ci.org/remind101/pkg) [![GoDoc](https://godoc.org/github.com/remind101/pkg?status.svg)](https://godoc.org/github.com/remind101/pkg)
 
-package pkg is a collection of Go packages that provide a layer of convenience over the stdlib and primarily adds **[context.Context](https://godoc.org/golang.org/x/net/context)** awareness, making it easier to do things like distributed request tracing.
+package pkg is a collection of Go packages that provide a layer of convenience over the stdlib and primarily adds **[context.Context](https://godoc.org/context)** awareness, making it easier to do things like distributed request tracing.
```

Comment:
This comment doesn't make a lot of sense anymore since there will be no need to add `context.Context` awareness after this transition is done.

Reply:
True, I'll remove that